### PR TITLE
DOC-2082-1 update node management console with expected behavior when…

### PR DESCRIPTION
**Pull request by erinking-r3**
… stopping the node

### CODE REVIEW

**Review by pmck91 — APPROVED** [review]

File: content/en/docs/corda-enterprise/4.7/node/management-console/index.md [hunk 1]
```diff
@@ -207,6 +207,10 @@ When the node management console is reloaded, the node diagnostic info is unavai
 * The buttons and toggle do not display.
 * The unavailable node information shows as `---`.
 
+{% note %}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.7/node/management-console/index.md:210]:
```suggestion
{{% note %}}
```

File: content/en/docs/corda-enterprise/4.8/node/management-console/index.md [hunk 2]
```diff
@@ -207,6 +207,10 @@ When the node management console is reloaded, the node diagnostic info is unavai
 * The buttons and toggle do not display.
 * The unavailable node information shows as `---`.
 
+{% note %}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.8/node/management-console/index.md:210]:
```suggestion
{{% note %}}
```

**Review by taylorharvey-r3 — COMMENTED** [review]
Overall looks good, just added suggestions to fix the short codes.

File: content/en/docs/corda-enterprise/4.7/node/management-console/index.md [hunk 3]
```diff
@@ -207,6 +207,10 @@ When the node management console is reloaded, the node diagnostic info is unavai
 * The buttons and toggle do not display.
 * The unavailable node information shows as `---`.
 
+{{% note %}}
+If you stop a node from the command line, the node will still appear in the console because Corda does not poll the node to see if it is still connected. Refresh the page to show the correct status.
+{{% /note%}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.7/node/management-console/index.md:212]:
```suggestion
{{% /note %}}
```

File: content/en/docs/corda-enterprise/4.8/node/management-console/index.md [hunk 4]
```diff
@@ -207,6 +207,10 @@ When the node management console is reloaded, the node diagnostic info is unavai
 * The buttons and toggle do not display.
 * The unavailable node information shows as `---`.
 
+{{% note %}}
+If you stop a node from the command line, the node will still appear in the console because Corda does not poll the node to see if it is still connected. Refresh the page to show the correct status. 
+{{% /note%}}
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.8/node/management-console/index.md:212]:
```suggestion
{{% /note %}}
```

**Review by taylorharvey-r3 — APPROVED** [review]
LGTM

**Pull request merged**